src/codeflare_sdk/cluster/cluster.py

```diff
@@ -600,9 +551,7 @@ def list_all_queued(namespace: str, print_to_console: bool = True):
     Returns (and prints by default) a list of all currently queued-up AppWrappers
     in a given namespace.
     """
-    app_wrappers = _get_app_wrappers(
-        namespace, filter=[AppWrapperStatus.RUNNING, AppWrapperStatus.PENDING]
-    )
+    app_wrappers = _get_app_wrappers(namespace, filter=[AppWrapperStatus.SUSPENDED])
```

Comment:
Shouldn't this still have `RUNNING` or do the new and old `RUNNING` statuses have different meanings?

Reply:
I wasn't sure what the desired semantics really were here.  The function is called `list_all_queued` and the comment says
```
    """
    Returns (and prints by default) a list of all currently queued-up AppWrappers
    in a given namespace.
    """
```
Neither old or new AppWrapper should be including `Running` in the filter for this, because Running means that the AppWrapper has been dispatched (it is not in the queue) and it is executing.  If we really did intend to include Running, then we should change the comment (and probably the name of the function).

Reply:
++, cant argue with you on that one especially since it's not used internally at all. I think your implementation is more clear w.r.t. implied functionality.